packages/dds/tree/src/feature-libraries/flex-tree/lazyField.ts

```diff
@@ -599,14 +602,14 @@ export class LazyForbiddenField<TTypes extends FlexAllowedTypes> extends LazyFie
 
 type Builder = new <TTypes extends FlexAllowedTypes>(
 	context: Context,
-	// TODO: use something other than `any`
+	// Correct use of these builders requires the builder of the matching type to be used.
+	// Since this has to be done at runtime anyway, trying to use safer typing then `any`  here (such as `never`, which is only slightly safer)
```

Comment:
```suggestion
	// Since this has to be done at runtime anyway, trying to use safer typing than `any` here (such as `never`, which is only slightly safer)
```